static/css/sb-admin-2.css

```diff
@@ -12011,3 +12011,23 @@ footer.sticky-footer .copyright {
 body.sidebar-toggled footer.sticky-footer {
   width: 100%;
 }
+
+.card-header {
+    background-color: rgb(245,245,245,0.3)!important;
+  border-radius:0!important;
+}
+
+.card{
+border-radius:0!important;
```

Comment:
nit: why are you using important! here?

Reply:
To be sure that the component that is using this class overrides all styles that can overlap with this one.